pkgs/kde/misc/kup/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, mkKdeDerivation
+, fetchFromGitLab
+, libgit2
+}:
+mkKdeDerivation {
+  pname = "kup";
+  version = "0.10.0";
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "system";
+    repo = "kup";
+    rev = "130face33a7500b6f881cf8dc7114a2ba2ea1081";
+    hash = "sha256-G/GXmcQI1OBnCE7saPHeHDAMeL2WR6nVttMlKV2e01I=";
+  };
+
+  buildInputs = [ libgit2 ];
+
+  meta = with lib; {
+    description = "Backup tool for KDE";
```

Comment:
This should get autofilled from the project data, if it exists

Reply:
Fixed